ansible_collections/arista/cvp/galaxy.yml

```diff
@@ -0,0 +1,56 @@
+### REQUIRED
+
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric characters and underscores. Additionally namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: arista
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: cvp
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.0
```

Comment:
should this be in sync with GH releases?

Reply:
Solved with PR #88